homeassistant/components/dynalite/__init__.py

```diff
@@ -1,5 +1,19 @@
 """Support for the Dynalite networks."""
-from dynalite_devices_lib import BRIDGE_CONFIG_SCHEMA
+from dynalite_devices_lib import (
+    CONF_ACTIVE,
```

Comment:
It's precarious to use the device library constants for the home assistant config keys. If the library changes we would have to make a breaking change in home assistant. Please decouple the config keys from the library.

Reply:
fixed